============================================================================================== 
Warning! Mixing Conda and module environments may lead to corruption of the
user environment. 
We do not recommend users mixing those two environments unless absolutely
necessary. Note that 
SURF does not provide any support for Conda environment.
For more information, please refer to our software policy page:
https://servicedesk.surf.nl/wiki/display/WIKI/Software+policy+Snellius#SoftwarepolicySnellius-UseofAnacondaandMinicondaenvironmentsonSnellius 

Remember that many packages have already been installed on the system and can
be loaded using 
the 'module load <package__name>' command. If you are uncertain if a package is
already available 
on the system, please use 'module avail' or 'module spider' to search for it.
============================================================================================== 
Current branch set-up-pipeline is up to date.
Obtaining file:///gpfs/home6/jhuizing/master-thesis-ds/ptdraft
  Installing build dependencies: started
  Installing build dependencies: finished with status 'done'
  Checking if build backend supports build_editable: started
  Checking if build backend supports build_editable: finished with status 'done'
  Getting requirements to build editable: started
  Getting requirements to build editable: finished with status 'done'
  Preparing editable metadata (pyproject.toml): started
  Preparing editable metadata (pyproject.toml): finished with status 'done'
Building wheels for collected packages: sisa3d
  Building editable for sisa3d (pyproject.toml): started
  Building editable for sisa3d (pyproject.toml): finished with status 'done'
  Created wheel for sisa3d: filename=sisa3d-0.1.0-py3-none-any.whl size=1286 sha256=466689da9623440a694956c93a4920038d607a9428abb993da0c7549d07f4ef9
  Stored in directory: /gpfs/scratch1/nodespecific/gcn48/jhuizing.6670304/pip-ephem-wheel-cache-dlzpksbc/wheels/ba/24/02/45c17a515af0a22326616b14da79c51e525e740ad6f0b007df
Successfully built sisa3d
Installing collected packages: sisa3d
  Attempting uninstall: sisa3d
    Found existing installation: sisa3d 0.1.0
    Uninstalling sisa3d-0.1.0:
      Successfully uninstalled sisa3d-0.1.0
Successfully installed sisa3d-0.1.0
[INFO] Creating temporary directory for config files: tmp_configs_330c1af3-4021-4800-aa18-9e26ac07d7ef
Generated 1 configuration files in tmp_configs_330c1af3-4021-4800-aa18-9e26ac07d7ef for 1 images.
[INFO] ✅ Temporary directory created with config files: tmp_configs_330c1af3-4021-4800-aa18-9e26ac07d7ef
[INFO] Running sequantial grid search with 1 configuration files...
Running grid search:   0%|          | 0/1 [00:00<?, ?it/s][INFO] Running pipeline with config: tmp_configs_330c1af3-4021-4800-aa18-9e26ac07d7ef/temporary_0_for_img_0.config.yaml
[INFO] Loading configuration...
[INFO] ✅ Configuration loaded.
[INFO] Loading paths...
[INFO] ✅ Paths loaded.
[INFO] Creating paths if they don't exist...
[INFO] ✅ Paths created.
[INFO] Initializing preprocessing pipeline...
[INFO] Running preprocessing pipeline...
[INFO] ✅ Preprocessing pipeline complete.
[INFO] Running DreamGaussian Stage 1 pipeline...
[INFO] ✅ DreamGaussian pipeline complete.
[INFO] Running Evaluation pipeline...
  0%|          | 0/1 [00:00<?, ?it/s]100%|██████████| 1/1 [00:01<00:00,  1.35s/it]100%|██████████| 1/1 [00:01<00:00,  1.37s/it]
  0%|          | 0/1 [00:00<?, ?it/s]100%|██████████| 1/1 [00:00<00:00,  1.73it/s]100%|██████████| 1/1 [00:00<00:00,  1.64it/s][INFO] ... Results saved to results/compare/compare.csv.
[INFO] ✅ Evaluation pipeline complete.
[INFO] Running DreamGaussian Stage 2 pipeline...
[INFO] Saving images of mesh...
[ERROR] ❌ Subprocess error with non-zero exit status 1
[ERROR] Standard error of the subprocess: Traceback (most recent call last):
  File "/home/jhuizing/.conda/envs/dreamgaussianV2/bin/kire", line 8, in <module>
    sys.exit(main())
  File "/home/jhuizing/.conda/envs/dreamgaussianV2/lib/python3.8/site-packages/kiui/render.py", line 397, in main
    gui = GUI(opt)
  File "/home/jhuizing/.conda/envs/dreamgaussianV2/lib/python3.8/site-packages/kiui/render.py", line 47, in __init__
    self.mesh = Mesh.load(opt.mesh, front_dir=opt.front_dir)
  File "/home/jhuizing/.conda/envs/dreamgaussianV2/lib/python3.8/site-packages/kiui/mesh.py", line 85, in load
    mesh = cls.load_obj(path, **kwargs)
  File "/home/jhuizing/.conda/envs/dreamgaussianV2/lib/python3.8/site-packages/kiui/mesh.py", line 167, in load_obj
    with open(path, "r") as f:
FileNotFoundError: [Errno 2] No such file or directory: 'results/mesh/stage_2/bird_rgba_mesh.obj'


Traceback (most recent call last):
  File "pipeline.py", line 221, in <module>
    raise cpe
  File "pipeline.py", line 171, in <module>
    result = subprocess.run(command, check=True, text=True, capture_output=True)
  File "/home/jhuizing/.conda/envs/dreamgaussianV2/lib/python3.8/subprocess.py", line 516, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['kire', 'results/mesh/stage_2/bird_rgba_mesh.obj', '--save', 'results/compare/stage2/mesh_images/ours//bird/', '--wogui']' returned non-zero exit status 1.
[ERROR] ❌ Subprocess error with non-zero exit status 1 for config: tmp_configs_330c1af3-4021-4800-aa18-9e26ac07d7ef/temporary_0_for_img_0.config.yaml
Running grid search:   0%|          | 0/1 [01:44<?, ?it/s]
Traceback (most recent call last):
  File "grid_search.py", line 69, in <module>
    run_pipeline(config_path)
  File "grid_search.py", line 38, in run_pipeline
    raise cpe
  File "grid_search.py", line 30, in run_pipeline
    subprocess.run(["python", "pipeline.py", "--config", config_path], check=True)
  File "/home/jhuizing/.conda/envs/dreamgaussianV2/lib/python3.8/subprocess.py", line 516, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['python', 'pipeline.py', '--config', 'tmp_configs_330c1af3-4021-4800-aa18-9e26ac07d7ef/temporary_0_for_img_0.config.yaml']' returned non-zero exit status 1.
[INFO] Creating temporary directory for config files: tmp_configs_b5faa049-ea58-4953-96ce-c52027250d67
Generated 1 configuration files in tmp_configs_b5faa049-ea58-4953-96ce-c52027250d67 for 1 images.
[INFO] ✅ Temporary directory created with config files: tmp_configs_b5faa049-ea58-4953-96ce-c52027250d67
[INFO] Running sequantial grid search with 1 configuration files...
Running grid search:   0%|          | 0/1 [00:00<?, ?it/s][INFO] Running pipeline with config: tmp_configs_b5faa049-ea58-4953-96ce-c52027250d67/temporary_0_for_img_0.config.yaml
[INFO] Loading configuration...
[INFO] ✅ Configuration loaded.
[INFO] Loading paths...
[INFO] ✅ Paths loaded.
[INFO] Creating paths if they don't exist...
[INFO] ✅ Paths created.
[INFO] Initializing preprocessing pipeline...
[INFO] Running preprocessing pipeline...
[INFO] ✅ Preprocessing pipeline complete.
[INFO] Running DreamGaussian Stage 1 pipeline...
[INFO] ✅ DreamGaussian pipeline complete.
[INFO] Running Evaluation pipeline...
  0%|          | 0/1 [00:00<?, ?it/s]100%|██████████| 1/1 [00:01<00:00,  1.40s/it]100%|██████████| 1/1 [00:01<00:00,  1.43s/it]
  0%|          | 0/1 [00:00<?, ?it/s]100%|██████████| 1/1 [00:00<00:00,  1.40it/s]100%|██████████| 1/1 [00:00<00:00,  1.34it/s][INFO] ... Results saved to results/compare/compare.csv.
[INFO] ✅ Evaluation pipeline complete.
[INFO] Running DreamGaussian Stage 2 pipeline...
[INFO] Saving images of mesh...
[ERROR] ❌ Subprocess error with non-zero exit status 1
[ERROR] Standard error of the subprocess: Traceback (most recent call last):
  File "/home/jhuizing/.conda/envs/dreamgaussianV2/bin/kire", line 8, in <module>
    sys.exit(main())
  File "/home/jhuizing/.conda/envs/dreamgaussianV2/lib/python3.8/site-packages/kiui/render.py", line 397, in main
    gui = GUI(opt)
  File "/home/jhuizing/.conda/envs/dreamgaussianV2/lib/python3.8/site-packages/kiui/render.py", line 47, in __init__
    self.mesh = Mesh.load(opt.mesh, front_dir=opt.front_dir)
  File "/home/jhuizing/.conda/envs/dreamgaussianV2/lib/python3.8/site-packages/kiui/mesh.py", line 85, in load
    mesh = cls.load_obj(path, **kwargs)
  File "/home/jhuizing/.conda/envs/dreamgaussianV2/lib/python3.8/site-packages/kiui/mesh.py", line 167, in load_obj
    with open(path, "r") as f:
FileNotFoundError: [Errno 2] No such file or directory: 'results/mesh/stage_2/bird_rgba_mesh.obj'


Traceback (most recent call last):
  File "pipeline.py", line 221, in <module>
    raise cpe
  File "pipeline.py", line 171, in <module>
    result = subprocess.run(command, check=True, text=True, capture_output=True)
  File "/home/jhuizing/.conda/envs/dreamgaussianV2/lib/python3.8/subprocess.py", line 516, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['kire', 'results/mesh/stage_2/bird_rgba_mesh.obj', '--save', 'results/compare/stage2/mesh_images/dreamgaussian//bird/', '--wogui']' returned non-zero exit status 1.
[ERROR] ❌ Subprocess error with non-zero exit status 1 for config: tmp_configs_b5faa049-ea58-4953-96ce-c52027250d67/temporary_0_for_img_0.config.yaml
Running grid search:   0%|          | 0/1 [01:30<?, ?it/s]
Traceback (most recent call last):
  File "grid_search.py", line 69, in <module>
    run_pipeline(config_path)
  File "grid_search.py", line 38, in run_pipeline
    raise cpe
  File "grid_search.py", line 30, in run_pipeline
    subprocess.run(["python", "pipeline.py", "--config", config_path], check=True)
  File "/home/jhuizing/.conda/envs/dreamgaussianV2/lib/python3.8/subprocess.py", line 516, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['python', 'pipeline.py', '--config', 'tmp_configs_b5faa049-ea58-4953-96ce-c52027250d67/temporary_0_for_img_0.config.yaml']' returned non-zero exit status 1.

JOB STATISTICS
==============
Job ID: 6670304
Cluster: snellius
User/Group: jhuizing/jhuizing
State: FAILED (exit code 1)
Nodes: 1
Cores per node: 18
CPU Utilized: 00:04:20
CPU Efficiency: 6.66% of 01:05:06 core-walltime
Job Wall-clock time: 00:03:37
Memory Utilized: 2.22 GB
Memory Efficiency: 1.85% of 120.00 GB
